ArmoniK.Extension.CSharp.Client/Common/Domain/Blob/BlobState.cs

```diff
@@ -39,6 +44,94 @@ public record BlobState : BlobInfo
   ///   Current status of the blob.
   /// </summary>
   public BlobStatus Status { get; init; }
+
+  /// <summary>
+  ///   The ID of the Task that as submitted this result.
+  /// </summary>
+  public string CreatedBy { get; init; } = string.Empty;
+
+  /// <summary>
+  ///   The owner task ID.
+  /// </summary>
+  public string OwnerId { get; init; } = string.Empty;
+
+  /// <summary>
+  ///   ID of the data in the underlying object storage.
+  /// </summary>
+  public byte[] OpaqueId { get; init; } = [];
+
+  /// <summary>
+  ///   The size of the blob Data.
+  /// </summary>
+  public int Size { get; init; }
+
+  /// <summary>
+  ///   Whether the user is responsible for the deletion of the data in the underlying object storage.
+  /// </summary>
+  public bool ManualDeletion { get; init; }
+
+  /// <summary>
+  ///   Custom Equals method
+  /// </summary>
+  /// <param name="other">the BlobState to compare the instance to</param>
+  /// <returns>true when the 2 instance have the same value, false otherwise</returns>
```

Comment:
```suggestion
  /// <inheritdoc>
```
Isn't it enough ? (I'm fine with both)

Reply:
Done